app/src/main/java/uk/gov/onelogin/tokens/usecases/RemoveAllSecureStoreData.kt

```diff
@@ -1,34 +1,29 @@
 package uk.gov.onelogin.tokens.usecases
 
 import android.util.Log
-import androidx.fragment.app.FragmentActivity
 import javax.inject.Inject
+import javax.inject.Named
 import uk.gov.android.securestore.SecureStore
 import uk.gov.android.securestore.error.SecureStorageError
 import uk.gov.onelogin.tokens.Keys
 
 fun interface RemoveAllSecureStoreData {
     /**
      * Use case for removing all data from the secure store instance
-     *
-     * @param context Must be a FragmentActivity context (due to authentication prompt)
      */
-    operator fun invoke(
-        context: FragmentActivity
-    )
+    operator fun invoke()
 }
 
 class RemoveAllSecureStoreDataImpl @Inject constructor(
-    private val secureStore: SecureStore
+    @Named("Token")
+    private val tokenSecureStore: SecureStore
 ) : RemoveAllSecureStoreData {
-    override fun invoke(context: FragmentActivity) {
+    override fun invoke() {
         try {
-            secureStore.delete(
-                context = context,
+            tokenSecureStore.delete(
                 key = Keys.ACCESS_TOKEN_KEY
             )
-            secureStore.delete(
-                context = context,
+            tokenSecureStore.delete(
                 key = Keys.ID_TOKEN_KEY
```

Comment:
Do you need to remove the `PersistentSessionID` here?

Reply:
It's not a part of this ticket, but eventually yes